ckanext/xloader/views.py

```diff
@@ -14,7 +14,15 @@ def get_blueprints():
 
 @xloader.route("/dataset/<id>/resource_data/<resource_id>", methods=("GET", "POST"))
 def resource_data(id, resource_id):
-    return utils.resource_data(id, resource_id)
+    rows = request.args.get('rows')
```

Comment:
This is unrelated.

Reply:
I needed to merge master because of merge conflicts